core/modules/qdisp/MessageStore.cc

```diff
@@ -48,9 +51,9 @@ namespace qdisp {
 
 void MessageStore::addMessage(int chunkId, int code, std::string const& description, MessageSeverity severity) {
     if (code < 0) {
-        LOGF_ERROR("Add msg: %1% %2% %3%" % chunkId % code % description);
+        LOGS(_log, LOG_LVL_ERROR, "Add msg: " << chunkId << " " << code << " " << description);
     } else {
-        LOGF_DEBUG("Add msg: %1% %2% %3%" % chunkId % code % description);
+        LOGS(_log, LOG_LVL_DEBUG, "Add msg: " << chunkId << " " << code << " " << description);
```

Comment:
To avoid repetition:

    auto level = code < 0 ? LOG_LVL_ERROR : LOG_LVL_DEBUG;
    LOGS(_log, level, "Add msg: " << chunkId << " " << code << " " << description);